pymc3/distributions/distribution.py

```diff
@@ -88,8 +88,11 @@ def getattr_value(self, val):
         return val
 
     def _repr_latex_(self, name=None, dist=None):
```

Comment:
https://github.com/ipython/ipython/blob/master/IPython/core/formatters.py#L782

as chris pointed out, this is a magic method IPython looks for